src/Runner.Worker/ActionRunner.cs

```diff
@@ -110,6 +104,13 @@ public async Task RunAsync()
                 localActionContainerSetupSteps = prepareResult.ContainerSetupSteps;
             }
 
+            if (handlerData.HasPre &&
```

Comment:
was this moved because it needs to come after prepare ActionsAsync when composite-local?

Reply:
correct, we need to load the local  composite action to know if it has a Pre